src/sel-to-where.js

```diff
@@ -1,16 +1,20 @@
+const {ATTRIBUTE_PRESENT} = require('./is-where');
+
 const selToWhere = sel => {
+  if (typeof sel === 'object') {
+    return sel;
+  }
   if (/^\./.test(sel)) {
     return {attributes: {class: sel.substring(1)}};
   }
   else if (/^#/.test(sel)) {
     return {attributes: {id: sel.substring(1)}};
   }
   else if (/^\[/.test(sel)) {
-    return {attributes: {[sel.substring(1, sel.length - 1)]: null}};
+    return {attributes: {[sel.substring(1, sel.length - 1)]: ATTRIBUTE_PRESENT}};
```

Comment:
particular note @mzgoddard I added a unique const to support the "Attribute present" wildcard instead of giving `null` a special meaning, helps support `.find(<Component attr={null} />)`

Reply:
Yeah. This is one of my favorite changes in this PR.